public/app/features/admin/UserListAdminPage.tsx

```diff
@@ -82,26 +83,28 @@ const UserListAdminPageUnConnected = ({
   return (
     <Page.Contents>
       <div className="page-action-bar" data-testid={selectors.container}>
-        <div className="gf-form gf-form--grow">
-          <FilterInput
-            placeholder="Search user by login, email, or name."
-            autoFocus={true}
-            value={query}
-            onChange={changeQuery}
-          />
-          <RadioButtonGroup
-            options={[
-              { label: 'All users', value: false },
-              { label: 'Active last 30 days', value: true },
-            ]}
-            onChange={(value) => changeFilter({ name: 'activeLast30Days', value })}
-            value={filters.find((f) => f.name === 'activeLast30Days')?.value}
-            className={styles.filter}
-          />
-          {extraFilters.map((FilterComponent, index) => (
-            <FilterComponent key={index} filters={filters} onChange={changeFilter} className={styles.filter} />
-          ))}
-        </div>
+        <InlineField grow>
+          <div>
+            <FilterInput
+              placeholder="Search user by login, email, or name."
+              autoFocus={true}
+              value={query}
+              onChange={changeQuery}
+            />
+            <RadioButtonGroup
+              options={[
+                { label: 'All users', value: false },
+                { label: 'Active last 30 days', value: true },
+              ]}
+              onChange={(value) => changeFilter({ name: 'activeLast30Days', value })}
+              value={filters.find((f) => f.name === 'activeLast30Days')?.value}
+              className={styles.filter}
+            />
+            {extraFilters.map((FilterComponent, index) => (
+              <FilterComponent key={index} filters={filters} onChange={changeFilter} className={styles.filter} />
+            ))}
+          </div>
+        </InlineField>
```

Comment:
Still an issue with the layout here. If you don't want to look into fixing it, I can accept the rest of the PR.

Reply:
My guess would be to only wrap the FilterInput

Reply:
Hello @tskarhed, I'll drop this file - will look into it with other PR when I raise it. Thanks